microsoft/testsuites/mshv/mshv_root_tests.py

```diff
@@ -145,13 +145,14 @@ def _mshv_stress_vm_create(
             node.tools[Free].log_memory_stats_mb()
             procs = []
             for i in range(vm_count):
+                vm_disk_img_path = node.working_path / f"VM{i}_{self.DISK_IMG_NAME}"
+                node.tools[Cp].copy(disk_img_path, vm_disk_img_path)
```

Comment:
are these vm disk files required after the testcase is finished? If not, we can clean them up and save some space on the Node.

Reply:
Done.